app/tests/unit/components/Form/ProjectEmissionIntensityReportFormSummary.test.tsx

```diff
@@ -203,11 +210,11 @@ describe("the emission intensity report form component", () => {
                   rowId: 2,
                   isPristine: false,
                   newFormData: {
+                    baselineEmissionIntensity: 0.87654321,
+                    targetEmissionIntensity: 0,
                     teimpReporting: {
                       baselineEmissionIntensity: 0.87654321,
                       targetEmissionIntensity: 0,
-                      postProjectEmissionIntensity: null,
-                      totalLifetimeEmissionReduction: null,
                     },
```

Comment:
Summary component has it but we don't extract its properties because we don't directly deal with it and simply pass the `newFormData` to the form component.